### PR TITLE
[15.0][ADD] purchase_order_invoicing_grouping_criteria: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa  TT40959

ping @carlosdauden @chienandalu

### CODE REVIEW

File: purchase_order_invoicing_grouping_criteria/README.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+For creating new grouping criteria:
+
+#. Go to *Invoicing > Configuration > Management > Invoicing Grouping Criteria*.
```
Comment (chienandalu) [inline purchase_order_invoicing_grouping_criteria/README.rst]: Purchase Invoicing Grouping Criteria?

**Review by chienandalu — APPROVED** [review]
Works as expected. Minor comment:

**Review by carlosdauden — APPROVED** [review]

File: purchase_order_invoicing_grouping_criteria/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply purchase_order_invoicing_grouping_criteria/README.rst]: @sergio-teruel are you changing this?

Comment (pedrobaeza) [reply purchase_order_invoicing_grouping_criteria/README.rst]: @sergio-teruel please change this for not getting confused with the sales invoicing grouping criteria.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**